packages/bot-web-ui/src/components/quick-strategy/quick-strategy.scss

```diff
@@ -395,4 +448,24 @@
             justify-content: center;
         }
     }
+    &__checkbox {
+        height: var(--input-height);
+        background: var(--general-main-1);
+        border-radius: 0.4rem;
+        display: flex;
+        align-items: center;
+        &__container {
+            display: flex;
+            align-items: center;
```

Comment:
maybe it would be better to use:  
```
@mixin align-center {
    display: flex;
    align-items: center;
}
```
There are 7 matches in this file.
Also since we have a few of the same mixins in different files maybe it would be better to take out them in one centralized file, wouldn't it? 
let's use more of the advanced stuff SASS✊

Reply:
Done 👍

Reply:
Instead of using mixing I used @extend which come more handy for scoped scss